### ISSUE TITLE
The Community Garden Network

### ISSUE THREAD

**Issue by jpoles1** [sketch-city/project-ideas#47]
This idea came out of a discussion at last month's health themed civic hack night:

A site which can be used to find community gardens near your home. Would utilize a community-submitted database of gardens (information including address, basic description, and who to contact in order to participate). 

I also envision a social networking component which allows users to join a talk-page for each garden, in order to coordinate and share important information.

**Comment by juyeongkim** [comment]
Hey @jpoles1! Here is my take on this project: https://houston.shinyapps.io/gardens/

I used Google Forms and R Markdown Flexboard to build this. It works for me because I don't know much about web development, but it's pretty slow since it's running the shiny instance every time it's loaded. I was wondering if I should just build the static page daily and upload it automatically instead. Also, maybe adding a comment section (maybe disqus?) to the dashboard.

**Comment by rlgreen91** [comment]
Hmm, I remember several people taking pictures of all of the ideas that we listed on the whiteboard.  Unfortunately, I don't have a copy of those.  @fileunderjeff , is there a way for us to get the pictures from the other organizers?

**Comment by jpoles1** [comment]
Hiya @juyeongkim, this is a great implementation of the idea; it looks good and works well!

I agree that the Shiny back-end is a bit slow, but I think that this can be fixed by either your static-page solution, or the use of a more web-friendly backend like PHP or Node.js and a suitable database (MongoDB or a SQL variant). Features like [searching](https://developers.google.com/maps/articles/phpsqlsearch_v3#creating-the-table) for nearby locations might be helpful as well.

I would like to work more on this project (I can help out on the web development side of things), but will not have time to do so until after May 20th.

**Comment by fileunderjeff** [comment]
@rlgreen91 was that at the sustainability hack night?

**Comment by jpoles1** [comment]
@fileunderjeff, this was an idea that came out of the health hack night.

**Comment by fileunderjeff** [comment]
Thanks @jpoles1! I am following up to find out who has pics.

**Comment by fileunderjeff** [comment]
Pics: https://www.facebook.com/media/set/?set=a.895240400590516.1073741835.297104993737396&type=3

Notes: Population Health BYOBrainstorm pitches:

HOUSTON HELPS- how can houston help- nonprofits
    meet dr but don’t know how to follow up
    see lots of non profits in ERs
    DATABASE of nonprofits in city- can’t afford to have someone in ER
    phone banks- on hold; out of date info

chronic conditions
    app- what is condition- personalized system
    reminders about visits and schedules
    personalized and precision- medication and well being
    dr.s know if patients are complying

we hate moquitos
    travel based zika (also dengue, chicken guinuea, malaria)
    mobilize high school sports (using sports celebrities to motivate)
        clean up trash (water based accumulation)
    not as invasive as fogging or fumagating

diabetes- sugar consumption
    game- educate kids and adults about healthy choices
        maze- harder to get healthy food
        drink coke- feel bad; apple- fuel take goes up
        ties to smartwatch- more active in real life- fuel goes up in game
    tareted advertizing and data analytics

wellness- nutrition, activity
    stuff gets in the way of eating well and exercise
    how do we keep it consistent
    yelp for wellness
    where can you engage in healthy activity? MAP
        where is best jogging route
        where is safe place to walk
        where is healthy place to eat

crowdsourced reviews
    consierge for small fee- what you should do on healthy day
    badges and rewards for healthy day

wellness- facilities hard to access
    does your park have swings?
    city datasets
    map of resources for wellness/parks
    layers of resources- value park ameneties (pdf->interactive map)
    updated by city (bike routes/etc)

community gardens- healthy gardens for food deserts
    in isolated pots in city
    platform- where are community gardens
    now you want to make your own- build pride in your neighborhood
    tell people about it- come get fresh food
    recipies to use with the garden materials

patient activity tracker- post op bariatrics patient
    18mil patients- only 1% get surgery
    insurance doesn’t pay for it because it doesnt always work
    incentivise dr to monitor - better outcomes- insurance will pay for it if more consistent
    patient incentives by better results and more follow ups with dr
    flagged by system if an issue- promote exercise intervention

**Comment by philellsworth** [comment]
Hi @fileunderjeff and @rlgreen91, I posted pics of three of the projects from the Health/Civic Hack Night. The City Wellness, Yelp for Wellness, and Community Gardens ones. I had hoped to put together a mockup of a site incorporating these ideas, but ran out of time this week.

**Comment by jpoles1** [comment]
In the interest of coordinating/structuring this project, I've created a [Google Doc](https://docs.google.com/document/d/17zIbdQk--2utzLdkwjp1IMxMPIrMTqJ5r1nkyVHAo0A/edit?usp=sharing) where we can lay out our objectives and plan our work.

**Comment by jpoles1** [comment]
I was doing some research and I came across the following pages:
https://communitygarden.org/find-a-garden/?garden-search=77005&distance=5&sensor=true
http://www.greenthumbnyc.org/gardensearch.html

It seems that some sort of garden map already exists; the question now becomes is it worth it for us to develop a separate application for this purpose? Should we change our focus to something else?

**Comment by fileunderjeff** [comment]
@jpoles1 while I think a garden map is cool, I agree that it might already exist in a "good enough" form. What about a map that identifies plots of land good for community gardens? e.g. cleared lots, government-owned blight, etc.

**Comment by jpoles1** [comment]
I'm not quite sure about the legal/practical aspects of starting gardens on this kind of land, but a map of blight does exist:
http://www.houstontx.gov/fighthoustonblight/blightmap.html

**Comment by fileunderjeff** [comment]
@jpoles1 yes, the goal would be to identify lots that would be good candidates for community gardens, whatever that criteria might be.

**Comment by rlgreen91** [comment]
So, rather than having the website focus on identifying community gardens, it could focus on getting started with creating one?

**Comment by ec-ong5** [comment]
Those are both good resources, but I don't think they include all the Houston community gardens. Houston itself doesn't seem to have one central location where all of this information can be found so it could be beneficial to still do this and partner with the city to create such a resource. 

Focusing on getting started with creating a community is also a great idea!

**Comment by rlgreen91** [comment]
@ec-ong5 - I'm not sure that it would make sense to do all of the work with establishing a website that lists Houston gardens specifically, when those resources do for some and it seems fairly straightforward to add new gardens to their listings.  Perhaps a compromise would be to see to what extent we could integrate one of these into our own site, by linking to them for certain topics for example.
